widgets/src/components/App/App.js

```diff
@@ -156,6 +159,26 @@ function App() {
               />
             )}
           />
+          <Route
+            path="/SummaryCard/"
+            render={() => (
+              <SummaryCardContainer
+                pageCode={WIDGETS_CONFIG.SUMMARY_CARD.pageCode}
```

Comment:
summaryId will not change on a widget in a page?

Reply:
I was thinking on setting summaryId on the config page, but i'm not sure though. Correct me if I'm wrong. Can you please help me understand why it needs to be on props?

Reply:
It depends on how the page will behave and what is your end goal. If your widget should be updated without page refresh (e.g., if it is related to a task and you can navigate through the tasks using some controls - next/prev buttons), then passing these values via props would utilize React and would only update the parts necessary vs. updating the whole page. So, if the summaryId would only change when he whole page is changed - then there is no need for props. Also, I would assume, the summaryId will be changeable only when adding the widget to the page or when going to page editor and changing the widget configs

Reply:
From what I observe (and looking at PDA-85), this only involves with one given summary ID only so yeah I don't think this needs to be a prop